src/sql/workbench/parts/notebook/cellViews/textCell.component.ts

```diff
@@ -52,6 +52,13 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		}
 	}
 
+	@HostListener('document:keydown.escape', ['$event'])
```

Comment:
How about code cell? Do we want to keep consistent?

Reply:
Good idea. Added logic to unselect the current (active) cell as well for both code and text cells.